app/src/renderer/components/staking/ModalUnstake.vue

```diff
@@ -0,0 +1,108 @@
+<template lang="pug">
+  .modal-stake#modal-stake
+    //- Header
+    .stake-header
+      img.icon(class='stake-atom' src="~assets/images/cosmos-logo.png")
+      span.tm-modal-title Unstake
+      .tm-modal-icon.tm-modal-close(@click="close()")
+        i.material-icons close
+
+    //- To
+    tm-form-group.stake-form-group(
+      field-id='to' field-label='To')
+      tm-field#to(
+        readonly v-model="to")
+
+    //- Amount
+    tm-form-group.stake-form-group(
+      field-id='amount'
+      field-label='Amount'
+    )
+      tm-field#amount(
+        :max="maximum"
+        :min="0"
```

Comment:
add step=1

Reply:
it is still possible to unstake decimals. and step=1 is the default.